### DISCUSSION TITLE
Mirth's "new" Community Center

### DISCUSSION THREAD

**Discussion by twest-mirthconnect** [nextgenhealthcare/connect#5003] [Mirth Mockups]
Attached is a new Community mock-up that promised to begin to take shape in 2022. The general idea is to consolidate the wider interoperability space for integration engines around the world.

**Take a look and share your thoughts and comments here for the next few weeks.**

(Mock-up removed)

Something like this:

**Phase 1 (MVP)**: Open-Source submissions to be approved and included in the core product through "**community**" free to everyone interested + Basic Profile

Account Details, Security, Subscription, Training, My Links, Webinars, Purchase History, Technical Support, Additional information
My Apps and Extensions
Community FAQs
Certified Partner Info
**Phase 2:** Commercial submissions to be approved and included in the core product through "**community**" as a

paid extension, plug-in,
services, or tool.
**Phase 3**: Partnership spotlights and exclusive benefits through Mirth Connect: Technical Services, Training, VAR, Gov't, Enterprise, & OEM

**Phase 4**: Collaboration and Sandbox

**Comment by rjkroll** [comment]
Nice!   If it turns out to be, and works, as good is it does in the Mock-up, then I would be excited to use this Community feature for sure.

**Comment by twest-mirthconnect** [comment]
Thank you for the feedback. We hope to have this funded for the current Fiscal Year to enable the whole community to contribute to and support the world of interoperability.
